### ISSUE TITLE
 `operator >> ` to build edges rather than `|`

### ISSUE THREAD

**Issue by ryanolson** [nv-morpheus/SRF#77]
I think the Clara team is using a ` >> ` operator for denote an edge.  Since we don't have a way of chaining pipes, using a different operator might be more reasonable.

This has nothing to do with the review of this PR, but we should open an issue if we want to make this change.

_Originally posted by @ryanolson in https://github.com/nv-morpheus/SRF/pull/75#discussion_r907676552_

**Comment by drobison00** [comment]
Just to restate this proposal, as I understand it:

`|` for intra-node `operator` chaining and `>>` for inter-node `edge` creation.

```mermaid
flowchart LR
 Source-- >> ---Node["op_1 | op_2 | ... | op_n"];
 Node-- >> ---Sink;
```

If so, we do have compound edge creation (edge chaining) in the PySRF adapters when both data types are registered with Pybind11.

**Comment by ryanolson** [comment]
Yes, and to be even more specific, `|` for intra-node `rxcpp::operator` chaining
